gencert-private.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+# generate configuration file
+
+openssl_conf=$(mktemp)
+
+cat > "${openssl_conf}" <<EOF
+oid_section=spid_oids
+
+[ req ]
+default_bits=2048
+default_md=sha256
+distinguished_name=dn
+encrypt_key=no
+prompt=no
+req_extensions=req_ext
+
+[ spid_oids ]
+agid=1.3.76.16
+spid-privatesector-SP=1.3.76.16.4.3.1
+spid-publicsector-SP=1.3.76.16.4.2.1
+uri=2.5.4.83
+
```

Comment:
The `organizationIdentifier` (`2.5.4.97`) field is not defined for `1.0.x` versions of OpenSSL. I would suggest to manage it as in the following

https://github.com/italia/spid-compliant-certificates/blob/d709afe6e94e548d53691b9bf9b40283057c73c7/gencert-public.sh#L11-L17

then

https://github.com/italia/spid-compliant-certificates/blob/d709afe6e94e548d53691b9bf9b40283057c73c7/gencert-public.sh#L33-L35

and

https://github.com/italia/spid-compliant-certificates/blob/d709afe6e94e548d53691b9bf9b40283057c73c7/gencert-public.sh#L111-L113